src/main/java/org/apache/datasketches/Criteria.java

```diff
@@ -355,9 +355,9 @@ String desc(final float[] arr, final int low, final int high, final float v, fin
    * The call to compare index a and index b with the value v.
    * @param arr The underlying sorted array of float values
    * @param a the lower index of the current pair
-   * @param b the higer index of the current pair
+   * @param b the higher index of the current pair
    * @param v the float value to search for
-   * @return +1, which means we must search higher in the aray, or -1, whicn means we must
+   * @return +1, which means we must search higher in the array, or -1, which means we must
```

Comment:
There is another occurrence of `aray` on line 349, as well as a typo `whicn` on the same line.